build.rs

```diff
@@ -74,6 +85,23 @@ fn build(output: &str) {
     }
 }
 
+fn is_os_x() -> bool {
+    let exit_code = Command::new("sw_vers")
```

Comment:
looks quite hackish to me, but well... the other option would be to check the host triple in $HOST